settings.gradle

```diff
@@ -2,4 +2,4 @@ rootProject.name='rxandroid-root'
 
 include 'rxandroid'
 include 'rxandroid-framework'
-include 'sample-app'
+//include 'sample-app'
```

Comment:
Yes, intentional to try and get 0.24 released until we can figure out how to have the `sample-app` working with the latest build plugin.